Source/CombatExtended/CombatExtended/Comps/CompFragments.cs

```diff
@@ -46,14 +43,13 @@ private static IEnumerator FragRoutine(Vector3 pos, Map map, float height, Thing
                 GenSpawn.Spawn(projectile, cell, map);
 
                 projectile.canTargetSelf = true;
-                projectile.minCollisionDistance = 1f;
-                //TODO : Don't hardcode at FragmentShadowChance, make XML-modifiable
-                projectile.castShadow = (Rand.Value < FragmentShadowChance);
+                projectile.minCollisionDistance = 0f;
```

Comment:
I am unsure how this may affect fragments, as I am fearing for the worst case scenario: fragments can hit each other, therefore cause the actual fragment count to be smaller.